src/Raven.Server/Documents/Handlers/CollectionsHandler.cs

```diff
@@ -53,6 +58,56 @@ public Task GetCollectionDocuments()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/databases/*/collections/$", "DELETE")]
+        public Task Delete()
```

Comment:
There is also a method in this controller 'DeleteCollectionDocuments'. Maybe it is redundant now?